app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -129,26 +128,28 @@ private void fillDatabaseWithDemoData() throws IOException {
         final Contact c = new Contact("Joost", new SimpleKey("Webserver", "dnfh4nl4jknlkjnr4j34klnk3j4nl"));
         final Contact d = new Contact("Kilian", new SimpleKey("Webmail", "sdjnefiniwfnfejewjnwnkenfk32"));
         final Contact e = new Contact("Cornel", new SimpleKey("Awesomeness", "nr23uinr3uin2o3uin23oi4un234ijn"));
-
-        final List<Contact> contacts = database.getAllContacts();
-        for (final Contact contact: contacts) {
-            database.deleteContact(contact.getName());
+        final Contact f = new Contact("Baas", new SimpleKey("Awesomeness", "nr23uinr3uin2o3uin23oi4un234ijn"));
+        if (!database.getAllContacts().isEmpty()) {
+            database.deleteContact(a.getName());
+            database.deleteContact(b.getName());
+            database.deleteContact(c.getName());
+            database.deleteContact(d.getName());
+            database.deleteContact(e.getName());
```

Comment:
I would think that this gives errors. 'Baas' will be added every time and doesn't get deleted, and a,b,c,d and e get deleted even though they are not added